include/criterion/assert.h

```diff
@@ -149,7 +149,7 @@
 /**@}*/
 
 /**
- * @defgroup CommonBinAsserts Common binary assertions
+ * @defgroup CommonBinaryAsserts Common binary assertions
```

Comment:
This change conflicts with the WIP on the sphinx/doxygen harmonization, and doesn't have anything to do with documenting `cr_assume_*` macros.